pkg/bucketaws/setup.go

```diff
@@ -152,3 +171,10 @@ func Credcheck(sess *session.Session) {
 	}
 	logrus.Info("Credentials file read succesfully")
 }
+
+// Get the absolute download path of the file
+func getDownloadPath(key string) string {
```

Comment:
Can we keep this one under the files package?
It makes more sense to have it there since it's about handling paths for files.